### PR TITLE
Refactor/Unify contract witness panics

### LINKED ISSUE #179: Create common message constants/Delete useless panic message prefix
Change `panic("put: " + errMessageConstant)` => `panic(errMessageConstant)`.

Create a common `owner/alphabet witness check failed` error message and reuse it in all contracts.

### PR DESCRIPTION
**Pull request by carpawell**
Closes #179.

### CODE REVIEW

File: common/witness.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	// by owner of some assets but was not.
+	ErrOwnerWitnessFailed = "owner witness check failed"
+	// ErrWitnessFailed appears when method must be called
+	// using certain public key but was not.
+	ErrWitnessFailed = "witness check failed"
+)
+
+// PanicOnAlphabetWitnessFail checks witness of the passed caller.
+// Panics with ErrAlphabetWitnessFailed message on fail.
+func PanicOnAlphabetWitnessFail(caller []byte) {
```
Comment (fyrchik) [inline common/witness.go:19]: `common.CheckAlphabetWitness` is a better choice IMO, same for other functions.

**Review by fyrchik — APPROVED** [review]
Other than comment, LGTM

File: common/witness.go (hunk 1 above)
Comment (carpawell) [reply common/witness.go:19]: agree, renamed

**Review by alexvanin — APPROVED** [review]

**Comment by alexvanin** [comment]
Why draft, though?

**Comment by carpawell** [comment]
> Why draft, though?

found out that I also want to delete prefixed from `runtime.Log()` messages

deleted in https://github.com/nspcc-dev/neofs-contract/pull/184/commits/08e97b91bf9a5048e1c7e9e362ee6187ca98a4ec, now it is ready

**Pull request merged**